integrations/neo4j/metrics/src/main/resources/META-INF/services/javax.enterprise.inject.spi.Extension

```diff
@@ -13,6 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-yamlProperty : "yes"
```

Comment:
Not sure what happened here - is this filre removed? Why does git think it was moved?